schema/suricata.schema

```diff
@@ -339,7 +338,6 @@ type suricata.smb = record {
       user: string
     },
     changed: count,
-    status: string,
```

Comment:
I see that `assigned_ip` occurs twice at the same hierarchy, but not `status`. Where is the duplication here?

Reply:
34 lines above.

Reply:
Gotcha!